examples/data_preprocessing/police_killing/preprocessing_with_kb.ipynb

```diff
@@ -0,0 +1,1883 @@
+       "3  Joe Shuman  [Joe Shuman]   \n",
+       "4  Joe Shuman  [Joe Shuman]   \n",
+       "\n",
+       "                                              sample  label  \n",
+       "0  [ ] TARGET / Chicago Tribune Lake County Major...      0  \n",
+       "1  Round Lake police shooting Round Lake police s...      0  \n",
+       "2  PERSON shooting PERSON shooting TARGET / Chica...      0  \n",
+       "3  Scene of Round Lake police shooting Scene of R...      0  \n",
+       "4  involved shooting TARGET / Chicago Tribune The...      0  "
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "def get_dev_test_data(data_path: str) -> Union[pd.DataFrame, pd.DataFrame]:\n",
+    "    with open(os.path.join(data_path, \"test.json\"), 'r') as data:\n",
+    "        dev_test_data = [json.loads(line) for line in data]\n",
+    "    dev_test_sent_alter = pd.DataFrame(dev_test_data, columns = [\"name\", \"names_org\", \"sent_alter\", \"plabel\"]).rename(columns={\"sent_alter\": \"sample\", \"plabel\": \"label\"})\n",
+    "    df_dev = dev_test_sent_alter.sample(n = int(round((dev_test_sent_alter.shape[0]/100)*used_as_dev))).reset_index(drop = True)\n",
+    "    df_test = dev_test_sent_alter.drop(df_dev.index).reset_index(drop = True)\n",
+    "    return df_dev, df_test\n",
+    "\n",
+    "df_dev, df_test = get_dev_test_data(data_path)\n",
+    "df_test.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1.3 Get Data of the Knowledge Base"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The FE-Database contains information about people dying or experiencing violence in encounters with the police, including the name of the victim, a short description of the incident and the use of force (whether a person in an encounter was, for instance, killed by the police or committed suicide or died in an accident while the police was present). For us, only the entries about actual police killings (no suicide, no accidents etc.) are relevant.\n",
+    "\n",
+    "It can be downloaded as an [Excel spreadsheet from Fatal Encounters](https://docs.google.com/spreadsheets/d/1dKmaV_JiWcG8XBoRgP8b4e9Eopkpgt7FL7nyspvzAsE/edit#gid=0). After that, we want to exclude all the information that is not useful for our task. \n",
+    "\n",
+    "We only need the first sheet that contains the actual information. In this sheet, we only need the name of the victim and the column *Intended use of force (Developing)*. We exclude all entries that do not describe *Deadly Force*.\n",
+    "\n",
+    "In the end, we keep only the relevant names."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fe_database = pd.read_excel(os.path.join(data_path, \"FATAL ENCOUNTERS DOT ORG SPREADSHEET (See Read me tab).xlsx\"), sheet_name = 0)"
```

Comment:
This gives a ModuleNotFoundError: No module named 'openpyxl'. ImportError: Missing optional dependency 'openpyxl'.  Use pip or conda to install openpyxl.